pandas/tests/series/indexing/test_indexing.py

```diff
@@ -907,3 +907,18 @@ def test_head_tail(test_data):
     assert_series_equal(test_data.series.head(0), test_data.series[0:0])
     assert_series_equal(test_data.series.tail(), test_data.series[-5:])
     assert_series_equal(test_data.series.tail(0), test_data.series[0:0])
+
+
+def test_index_map():
+    # see GH20990
+    count = 6
+    index = pd.date_range("2018-01-01", periods=count, freq="M")
+
+    a = Series(np.arange(count), index=index)
+    a.index.name = "name"
+
+    b = a.copy()
+    try:
```

Comment:
Instead of this try / except, construct the expected result, and use `tm.assert_series_equal(b, expected)`